molecule/default_with-worker/molecule.yml

```diff
@@ -16,6 +16,7 @@ platforms:
       - /var/lib/containerd
     privileged: true
     pre_build_image: true
+    keep_volumes: false
```

Comment:
What this option actually does?

Reply:
By default docker plugin saves volumes after destroying:

[source](https://github.com/ansible-community/molecule-plugins/blob/7bb0b345f00a473a78d05c350acbf10f4e928957/src/molecule_plugins/docker/playbooks/destroy.yml#LL25C9-L25C21)